src/hooks/purchaseHistory.ts

```diff
@@ -21,14 +21,17 @@ export const usePurchaseHistory = (
     const asyncFetchData = async () => {
       setLoading(true);
       try {
-        const res = await fetch(`${CORETIME_API[network]}/broker/purchased`, {
-          method: 'POST',
-          body: JSON.stringify({
-            region_begin: regionBegin,
-            row,
-            page,
-          }),
-        });
+        const res = await fetch(
+          `${SUBSCAN_CORETIME_API[network]}/api/scan/broker/purchased`,
```

Comment:
We previously didn't have '/api/scan/' here; instead, we had it in the base URL, i.e., in `SUBSCAN_CORETIME_API`. I moved it here instead, but I'm not sure if this makes more sense. @TopETH, thoughts?